doc/source/examples/metrology/SOLT.ipynb

```diff
@@ -179,10 +175,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# generate ideal 2-ports, as required by the scikit-rf SOLT calibration Class \n",
-    "short_ideal_2p = rf.two_port_reflect(short_ideal, short_ideal)\n",
-    "open_ideal_2p = rf.two_port_reflect(open_ideal, open_ideal)\n",
-    "load_ideal_2p = rf.two_port_reflect(load_ideal, load_ideal)"
+    "# ideal 2-port Networks\n",
+    "short_ideal_2p = media.short(nports=2)\n",
+    "open_ideal_2p = media.open(nports=2)\n",
+    "load_ideal_2p = media.match(nports=2)  # could also be: media.load(nports=2, Gamma0=0)\n",
+    "thru_ideal_2p = media.thru()\n",
```

Comment:
pay attention that `thru_ideal_2p` is not used after, as the list of ideals uses `thru_ideal` instead. This leads to a failure of the notebook